.gitignore

```diff
@@ -1 +1,2 @@
 /target
+examples/*/target
```

Comment:
Not needed.

```suggestion
```

Reply:
Then we need to change `/target` to `target`, without it:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        examples/std-hello-world/target/

```